Tests/JaegerTests/TestUtilities.swift

```diff
@@ -0,0 +1,113 @@
+//
+//  TestUtilities.swift
+//  JaegerTests
+//
+//  Created by Simon-Pierre Roy on 10/30/18.
+//
+
+import XCTest
+@testable import Jaeger
+
+/**
+Useful and reusable constants to help the construction of new tests.
+ */
+enum TestUtilitiesConstants {
+    /// Fixed UUID for a `Span`
+    static let spanUUID = UUID(uuidString: "271C452F-D78A-4612-9425-79BCC21B3811")!
+    /// Fixed UUID for a `Trace`
+    static let traceUUID = UUID(uuidString: "54186C03-8F55-403F-97D0-CF602CE3D903")!
+}
+
+/**
+ A stub Tracer.
+ 
+Useful to test Spans when reporting can be ignored. **Do not use** the `startSpan` function.
+ */
+class EmptyTestTracer: Tracer {
+    /// **Do not use**
+    func startSpan(operationName: String, references: Span.Reference?, startTime: Date, tags: [Tag]) -> OTSpan {
+        fatalError()
```

Comment:
is there a specific reason to call `fatalError` here vs doing nothing? If so, could you leave a comment saying why?

Reply:
I want this EmptyTestTracer to be used as a stub that can not produce spans. Since `startSpan` returns an `OTSpan` I am calling `fatalError()`.

Reply:
AH got it :)